test/rules/triple-equals/tslint.json

```diff
@@ -1,5 +1,5 @@
 {
   "rules": {
-    "triple-equals": [true, "allow-null-check"]
+    "triple-equals": [true, "allow-null-check", "allow-undefined-check"]
```

Comment:
Ideally in these tests, it would be better to make a separate test case for this option. See how the `semicolon` rule has its tests structured for example.